modules/virt-networking-glossary.adoc

```diff
@@ -16,10 +16,13 @@ project, focused on container network connectivity.
 
 Multus:: a "meta" CNI plug-in that allows multiple CNIs to exist so that a pod or virtual machine can use the interfaces it needs.
 
-Custom resource definition (CRD):: a link:https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/[Kubernetes]
+Custom Resource Definition (CRD):: a link:https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/[Kubernetes]
 API resource that allows you to define custom resources, or an object defined by using the CRD API resource.
 
-Network attachment definition:: a CRD introduced by the Multus project that allows you to attach pods, virtual machines, and virtual machine instances to one or more networks.
+Network Attachment Definition (NAD):: a CRD introduced by the Multus project that allows you to attach pods, virtual machines, and virtual machine instances to one or more networks.
+
+Node Network Configuration Policy (NNCP):: a description of the requested network configuration on nodes.
+You update the node network configuration, including adding and removing interfaces, by applying a `NodeNetworkConfigurationPolicy` manifest to the cluster.
```

Comment:
@mgarrellRH Though it might look inconsistent, the items in this glossary do not all get the same capitalization treatment. 
- General references to API objects (including the NAD and NNCP definitions) get sentence case as per our guidelines.
- [Container Network Interface](https://www.cni.dev/), IIUC, is a project title, so it gets title case. 
- The contrib guide has a [glossary entry for CRD](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#custom-resource-definition-crd), showing that it gets sentence case.